README.md

```diff
@@ -1,24 +1,28 @@
 
-The publishing of these datasets as well as the characterization of multiple feature groups across a broad population and multiple settings are intended to aid future cardiovascular research. These datasets include de-identified participant information, as well as aligned and featurized sensor data. See publication for more information. Below are details describing the published dataset and access information.
+Note that the data contained in this repository represent a very small sample of the full dataset, meant only to illustrate the structure of the files and allow testing with the sample code. For access to the full dataset, see the [Data Use Application](#data-use-application) section below.
 
 ## Navigation:
-- 'docs': 
-  - README with data file descriptions, a detailed Aurora-BP Study protocol overview, and supplemental results not included in the Aurora-BP Study Publication
-- 'notebooks':
-  - Sample Jupyter notebooks for basic analyses using Aurora-BP Study data
-- 'sample':
+- [docs](docs/README.md): 
+  - Data file descriptions, a detailed overview of the Aurora-BP Study protocol, and supplemental results not included in the Aurora-BP Study publication
+- [notebooks](notebooks/README.md):
+  - Sample Jupyter notebooks and environment files for basic analyses using Aurora-BP Study data
+- [sample](sample):
   - Example data files, to run sample Jupyter notebooks and provide researchers a direct look at the data format before application for full data access.
 
+## Citation
+If you use this repository, part or all of the full dataset, and/or our paper as part of your research, please refer to the dataset as the **Aurora-BP dataset** and cite the publication as below:
+- Mieloszyk, Rebecca, et al. "[A Comparison of Wearable Tonometry, Photoplethysmography, and Electrocardiography for Cuffless Measurement of Blood Pressure in an Ambulatory Setting.](https://ieeexplore.ieee.org/document/9721156)" *IEEE Journal of Biomedical and Health Informatics* (2022)
+
 ---
 
 ## Data Access
 
 ### Data Access Committee
 
-Requests for [data access](docs/README.md) are reviewed by the Data Access Committee. During review, submitting investigator and primary investigator may be contacted for verification. For further information regarding data access, contact: aurorabp@microsoft.com
+Requests for [data access](docs/README.md) are reviewed by the Data Access Committee. During review, the submitting investigator and primary investigator may be contacted for verification. The information you will need to gather for a submitting a Data Use Application as well as a link to the form are listed below. For additional questions regarding data access, contact: aurorabp@microsoft.com

```

Comment:
instead of "for a submitting" it should be "to submit" or "for a Data Use Application submission"

Reply:
good catch, fixing